### ISSUE TITLE
Add a cheatsheet image for color/UI settings

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#5672]
Basically a screenshot with most UI elements visible (completion, downloads, etc.) and callouts to the respective setting names.

**Comment by The-Compiler** [comment]
Another idea, though that'd require some adjustment of the documentation system (and deployment) as well: Have individual images in the docs for every single UI setting. Would be nice to have after #345 certainly!
